app/api/v1/redflags/views.py

```diff
@@ -0,0 +1,26 @@
+from flask_restful import Resource
+from flask import request, make_response, jsonify
+import json
+ 
+from .models import db
+
+
+class RedFlag(Resource):
+    def __init__(self):
+        self.db = db
+        self.id = len(db)+1
+
+    def get(self):
+        return db, 200
+        
+    def post(self):
```

Comment:
This should make use of data parsed from user request to create and object. As it is it is just returning a success message without actually creating an incident report